### ISSUE TITLE
Color contrast of link text with state changes

### ISSUE THREAD

**Issue by aviannamiller** [omeka/Omeka#1082]
The following themes have link text with problematic color contrast on hover and/or focus:

**Berlin**
- On hover on all link text and on both hover and/or focus in nav, contrast is in the 2.4:1-2.5:1 range (depending on the background)

**Foundation/Seafoam** (in respect to the horizontal dropdown nav layout only)
- Nav links on hover and/or focus within the submenu are at 4.3:1
- Parent menu items with child links on hover are 1:1

**Foundation/Inkwell** 
- Generally speaking, links on hover and/or focus are 2.1:1
- In respect to the horizontal dropdown nav layout only:
  - Submenu nav links on hover and/or focus are 1.7:1
  - Parent menu items with child links on hover and/or focus are 1.5:1

**Minimalist**
- Generally, links on hover are 3.2:1
- Submenu items on hover and/or focus  are at 2.3:1

Since all of the above stated are within parameters for regular text, contrast should be updated to at least 4.5:1.

**Comment by kimisgold** [comment]
The affected themes should all have fixes on their default branches.

**Comment by aviannamiller** [comment]
Changes look great, thank you!

For Minimalist: **visited** `.view-items-link`'s (Browse Collections) and visited links in the mobile nav menu are 4:1 with that grey background. Not sure if any other elements with that background color pick up on the visited style, but these are the only areas I see at the moment.

**Comment by kimisgold** [comment]
It shouldn't have been picking up that style, and should be fixed now.

**Comment by aviannamiller** [comment]
The view item links are improved, but I still see that low contrast visited style appearing on mobile `.nav-item` links.

**Comment by aviannamiller** [comment]
Minimalist: Also an issue with the following:
```
#exhibit-pages.full #secondary-nav ul li a:visited {
    color: #684B98;
}
```

This leads to a 1.2:1 contrast against the #7453a9 background

**Comment by kimisgold** [comment]
Thanks for catching those, I think I got them both.

**Comment by aviannamiller** [comment]
Great, thanks. That should be all on this then.
